src/main/resources/assets/app/scripts/models/base_job.js

```diff
@@ -21,7 +21,6 @@ function($, Backbone, _, moment, BaseJobValidations) {
         encoded;
 
     encoded = _.map(args, function(arg) { return encodeURIComponent(arg); });
-    encoded.unshift('');
```

Comment:
This is not an empty fail-safe argument.
The only reason this was added is to let the .join('/') in the line below add the '/', so the array gets something like this ['', ''] what is joined to ''+'/'+''.
Now with relative paths, we don't need this anymore and the line was removed.